src/ol/PluggableMap.js

```diff
@@ -1103,6 +1103,9 @@ class PluggableMap extends BaseObject {
     const view = this.getView();
     if (view) {
       this.viewport_.setAttribute('data-view', getUid(view));
```

Comment:
This line is now obsolete and can be removed. Also, the JSDoc for `ol/View#calculateExtent`'s optional `size` parameter can be changed now:

> If not provided, the size of the map that uses this view will be used.